### PR TITLE
Various tweaks to get this to deploy properly

### PR DESCRIPTION
**Pull request by imbstack**

### CODE REVIEW

File: src/build/utils.js [hunk 1]
```diff
@@ -57,23 +57,23 @@ exports.gitClone = async ({dir, url, sha, utils}) => {
...
+    if (existingRev === remoteRev) {
+      return {exactRev: existingRev, changed: false};
+    }
   }
 
-  await exec('git', ['fetch', repo, ref], opts);
-  await exec('git', ['reset', '--hard', 'FETCH_HEAD'], opts);
-  const exactRev = (await exec('git', ['rev-parse', 'HEAD'], opts)).stdout.split(/\s+/)[0];
-  return {exactRev, changed: true};
+  await exec('rm', ['-rf', dir]);
```
Comment (djmitche) [inline src/build/utils.js]: Maybe a comment as to why this is best?  (I think we talked about it in irc..)

File: src/build/utils.js [hunk 2]
```diff
@@ -130,8 +130,10 @@ exports.dockerRun = async ({baseDir, logfile, command, env, binds, workingDir, i
   const {docker, dockerRunOpts} = await _dockerSetup({baseDir});
 
   const output = new PassThrough();
+  let errorAddendum = '';
   if (logfile) {
     output.pipe(fs.createWriteStream(logfile));
+    errorAddendum = ` Logs available in ${logfile}`;
```
Comment (djmitche) [inline src/build/utils.js]: ++

File: src/formats/cluster-spec.js [hunk 3]
```diff
@@ -24,18 +24,23 @@ class ClusterSpec {
 
     const deployFile = path.join(filename, 'deploy.yml');
     this.deploy = yaml.safeLoad(fs.readFileSync(deployFile));
+
+    const ingressFile = path.join(filename, 'ingress.yml');
```
Comment (djmitche) [inline src/formats/cluster-spec.js]: This should be documented in `docs/cluster-spec.yml`

**Comment by imbstack** [comment]
I think eventually we can auto-generate the ingress, but I figured let's go fast for now.

**Pull request merged**